exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -259,7 +260,8 @@ func (e *exporter) sendBatch(ctx context.Context, lines []string) error {
 func (e *exporter) start(_ context.Context, host component.Host) (err error) {
 	client, err := e.cfg.HTTPClientSettings.ToClient(host.GetExtensions(), e.settings)
 	if err != nil {
-		return err
+		e.settings.Logger.Sugar().Errorf("failed to construct HTTP client %v", err)
```

Comment:
```suggestion
e.settings.Logger.Sugar().Error("Failed to construct HTTP client", zap.Error(err))
```
Same as above, keeping structured logging simple to make searching for errors simple :)